common/models/user.js

```diff
@@ -672,13 +672,18 @@ module.exports = function(User) {
       return process.nextTick(cb);
 
     var AccessToken = accessTokenRelation.modelTo;
-
     var query = {userId: {inq: userIds}};
     var tokenPK = AccessToken.definition.idName() || 'id';
     if (options.accessToken && tokenPK in options.accessToken) {
       query[tokenPK] = {neq: options.accessToken[tokenPK]};
     }
-
+    // add principalType in AccessToken.query if using polymorphic relations
+    // between AccessToken and User
+    var isTokenRelationToUserPolymorphic = AccessToken.relations.user.polymorphic &&
+    !AccessToken.relations.user.modelTo;
```

Comment:
i you refer to `!AccessToken.relations.user.modelTo` exclusively (no the entire isTokenRelationToUserPolymorphic definition) : 
let's say i'm not confident enough that AccessToken.relations.user.polymorphic (part of codebase i know a bit less) is always there so i am tinkering. 
-> if you believe testing the polymorphic object is enough, then i'm ok to remove that part.